src/goto-programs/goto_contractor.cpp

```diff
@@ -307,6 +367,13 @@ goto_contractort::create_function_from_expr2t(irep_container<expr2t> expr)
     f = new ibex::Function(*vars, c);
     break;
   }
+  case expr2t::expr_ids::constant_floatbv_id:
```

Comment:
Do we really support float?

Reply:
We don't yet. removed. I was trying to handle floats in this update, but I faced some problems with it.